{"uuid": "dbec4159-ba9d-4632-842f-8f36c236258e", "children": ["56cc5c8c-44ae-4dfa-8da0-bc605d7fdd8d"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'base_url'\n", "trace": "  File \"E:\\Python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 921, in call_fixture_func\n    fixture_result = next(generator)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 799, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 732, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\GitTest\\Pytest126mail\\Public\\SetUp_TearDown.py\", line 29, in setup_class\n    self.goto_url('',selenium_driver=self.driver)\n  File \"E:\\GitTest\\Pytest126mail\\Public\\Base.py\", line 11, in goto_url\n    self.base_url = base_url                             # 和超时时间（timeout）\n"}, "start": 1608736892038, "stop": 1608736892038}], "start": 1608736892038, "stop": 1608736892111}